src/convert.jl

```diff
@@ -930,3 +930,21 @@ function convert(::Type{MinkowskiSumArray},
                  X::MinkowskiSum{N, ST, MinkowskiSumArray{N, ST}}) where {N, ST}
     return MinkowskiSumArray(vcat(X.X, X.Y.array))
 end
+
+"""
+    convert(::Type{Interval}, x::MinkowskiSum{N, IT, IT}) where {N, IT<:Interval{N}}
+
+Converts the Minkowski sum of two intervals into an interval.
+
+### Input
+
+- `Interval` -- type used for dispatch
+- `X`        -- Minkowski sum of a pair of intervals
```

Comment:
```suggestion
- `x`        -- Minkowski sum of a pair of intervals
```
Although I would prefer to use capital letters for sets...